sdk/tests/conformance/context/context-size-change.html

```diff
@@ -70,10 +70,10 @@
     gl.clearColor(1.0, 1.0, 1.0, 1.0);
     gl.clear(gl.COLOR_BUFFER_BIT);
 
-    // Check that the top-left pixel has R channel 255.
+    // Check that the top-left pixel has R channel non-zero.
```

Comment:
Can this check be strengthened and still pass with CMAA? The idea that this test could pass while allowing that pixel to have a value of 1 out of 255 (still looking quite black) is undesirable.

Reply:
The actual pixel value after CMAA applied is 183. I am uncomfortable to use this magic value here, as the value may change if CMAA evolves or we use other algorithms like FXAA. Maybe it fair to be more than half of 255.
Another idea in my mind is that we should apply CMAA only if there was a draw to the framebuffer.  After all MSAA doesn't interfere with clear calls as well. Do you think we should seek to refine the current WebGL implementation regarding the timing of applying screen-space anti-alias?